x-pack/plugins/synthetics/public/apps/synthetics/state/synthetics_enablement/index.ts

```diff
@@ -39,6 +46,18 @@ export const syntheticsEnablementReducer = createReducer(initialState, (builder)
     .addCase(getSyntheticsEnablementFailure, (state, action) => {
       state.loading = false;
       state.error = action.payload;
+    })
```

Comment:
I talked about it above in the `useEffect` predicate responsible for dispatching this `get` action, but we have a potential for an unbound request loop with how we're assigning things here.